src/core/parser/PDFParser.ts

```diff
@@ -162,9 +164,17 @@ class PDFParser extends PDFObjectParser {
   }
 
   // TODO: Improve and clean this up
-  private tryToParseInvalidIndirectObject() {
+  private tryToParseInvalidIndirectObject(
+    throwOnInvalidObject: boolean = false,
+  ) {
     const startPos = this.bytes.position();
 
+    if (throwOnInvalidObject) {
```

Comment:
This can be cleaned up a bit to only occupy only 3 lines (instead of 9) and to avoid duplicating the error message:
```js
const msg = `Trying to parse invalid object: ${JSON.stringify(startPos)})`;
if (this.throwOnInvalidObject) throw new Error(msg);
console.warn(msg);
```

Reply:
Seems obvious now that I think about it